src/NLogTarget/NLogTarget.csproj

```diff
@@ -41,7 +41,7 @@
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.5.0-beta1" />
-    <PackageReference Include="NLog" Version="4.3.11" />
+    <PackageReference Include="NLog" Version="4.4.12" />
```

Comment:
Make sure https://github.com/Microsoft/ApplicationInsights-dotnet-logging/blob/develop/ThirdPartyNotices.txt is up to date license-wise. Also mention this in CHANGELOG as this update make 4.3 of NLog not supported.

Reply:
Done and done.